internal/daysteps/daysteps.go

```diff
@@ -10,8 +14,42 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	// разделяю строку
+	slice := strings.Split(data, ",")
+	// проверяю длину полученного слайса, вывожу ошибку, если она более 2
+	if len(slice) != 2 {
+		return 0, 0, fmt.Errorf("длина слайса больше 2")
+	}
+	// преобразую строку с шагами в int
+	steps, err := strconv.Atoi(slice[0])
+	// вывод ошибки, если преобразование неуспешное
+	if err != nil {
+		return 0, 0, fmt.Errorf("ошибка преобразования шагов из strings в int")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```